packages/amplify-opensearch-simulator/src/index.ts

```diff
@@ -347,3 +353,16 @@ export const getPathToOpenSearchBinary = async (pathToOpenSearchLocal?: string):
   }
   return join('opensearchLib', 'bin', 'opensearch');
 };
+
+export const getPackageAssetPaths: GetPackageAssetPaths = async () => [relativePathToOpensearchLocal];
```

Comment:
What files are being copied on CLI install?

Reply:
No files are copied, just the directory structure is created as `~/.amplify/lib/amplify-opensearch-simulator/opensearch/1.3.0`.